build.proj

```diff
@@ -79,6 +79,10 @@
     <TestFile Include="$(MSBuildThisFileDirectory)/test/data/external/winequality-white.csv"
       Url="https://archive.ics.uci.edu/ml/machine-learning-databases/wine-quality/winequality-white.csv"
       DestinationFile="$(MSBuildThisFileDirectory)test/data/external/winequality-white.csv" />
+    
+    <TestFile Include="$(MSBuildThisFileDirectory)/test/data/external/wikiDetoxAnnotated160kRows.tsv"
```

Comment:
We shouldn't be downloading this file unless we need to run perf tests. It is a rather large file to download. So we should probably put this behind a `Condition`.